src/libraries/tests.proj

```diff
@@ -311,7 +311,6 @@
 
     <!-- App Crash https://github.com/dotnet/runtime/issues/53624 -->
     <ProjectExclusions Include="$(MSBuildThisFileDirectory)Microsoft.CSharp/tests/Microsoft.CSharp.Tests.csproj" />
-    <ProjectExclusions Include="$(MSBuildThisFileDirectory)System.Diagnostics.Tracing/tests/System.Diagnostics.Tracing.Tests.csproj" />
```

Comment:
We also need to remove the ActiveIssues like https://github.com/dotnet/runtime/blob/bdc0d1a409a24d4092e7b108c13fc4ff1c643d1f/src/libraries/System.Diagnostics.Tracing/tests/BasicEventSourceTest/TestsManifestNegative.cs#L62